cmd/security-file-token-provider/res/token-config.json

```diff
@@ -0,0 +1,64 @@
+{
+  "edgex-config-seed": {
+    "edgex_use_defaults": true
+  },
+  "edgex-core-command": {
+    "edgex_use_defaults": true
+  },
+  "edgex-core-data": {
+    "edgex_use_defaults": true
+  },
+  "edgex-core-metadata": {
+    "edgex_use_defaults": true
+  },
+  "edgex-export-client": {
+    "edgex_use_defaults": true
+  },
+  "edgex-export-distro": {
+    "edgex_use_defaults": true
+  },
```

Comment:
Omit these.  I expect to merge https://github.com/edgexfoundry/edgex-go/pull/2219 later today; it will remove all traces of the export services from edgex-go.

Reply:
Removed.